codecov.yml

```diff
@@ -1,26 +1,18 @@
 coverage:
   status:
-    patch:
-      default:
-        target: 0%
-        threshold: 0%
-      core-packages:
-        target: 0%
-        paths:
-          - packages
-      plugins:
-        target: 0%
-        paths:
-          - plugins
+    patch: off
     project:
       default:
         target: 0%
-        # send notification if coverage change exceed this threshold
-        threshold: 0%
-      core-packages:
+        threshold: 10%
+      core-packages-ts:
         target: 100%
         paths:
-          - packages
+          - packages/**/*.{js|ts}$
+      core-packages-tsx:
+        target: 10%
```

Comment:
`100%` ?

Could we combine the `core-packages-ts` and `core-packages-tsx` section then add the `ignore` path for `control-utils`'s `{t|j}sx`?

Reply:
Probably not 100% since relaxing this constraint on `{j|t}sx` is the point. That said, the number can be negotiated/adjusted at any point. 50%? ¯\\\_(ツ)_/¯